### PR TITLE
Naxx Fix various Boss configurations

### PR DESCRIPTION
**Pull request by insunaa**
Increased the run speed of the 4hm to be similar to the reference client
Added leashes for all bosses requiring leashes (except maybe Maexxna)
Corrects incorrect Gluth spawn location and orientation
Increase detection range to more appropriate levels

Partially fixes https://github.com/cmangos/issues/issues/2387
Partially fixes https://github.com/cmangos/issues/issues/2389
Fixes https://github.com/cmangos/issues/issues/2388 in conjunction with https://github.com/cmangos/mangos-tbc/pull/426

ToDo:
- [x] Investigate if Maexxna needs a leash
- [x] Investigate Detection Range for Noth
- [x] Investigate Detection Range for Anub'Rekhan
- [x] Investigate Detection Range for Grobbulus

### CODE REVIEW

**Comment by cala** [comment]
Greetings and thank you @insunaa 
I see that there are still unchecked box in the todo list. 
Is this still a work in progress ?

File: Updates/0414_naxxramas_boss_fixes.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+UPDATE creature_template SET Leash=100 WHERE entry=16011;
+-- Noth the Plaguebringer
+UPDATE creature_template SET Leash=120 WHERE entry=15954;
+-- Anub'Rekhan
+UPDATE creature_template SET Leash=130 WHERE entry=15956;
+-- Grobbulus
+UPDATE creature_template SET Leash=180 WHERE entry=15931;
+-- Gluth
+UPDATE creature_template SET Detection=70 WHERE entry=15932;
+UPDATE creature SET position_x="3300.73", position_y="-3140.42", position_z="296.89", orientation="3.90" WHERE guid=5330146;
```
Comment (AnonXS) [inline Updates/0414_naxxramas_boss_fixes.sql:23]: update in instance file please

Comment (insunaa) [reply Updates/0414_naxxramas_boss_fixes.sql:23]: Will do, one sec.

**Comment by insunaa** [comment]
@cala this is as good as I can get it

File: Updates/0414_naxxramas_boss_fixes.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+-- Maexxna
+UPDATE creature_template SET SpeedRun=1.68, Detection=60 WHERE entry=15952;
+-- Patchwerk
+UPDATE creature_template SET Detection=40 WHERE entry=16028;
+-- Grobbulus
+UPDATE creature_template SET Detection=35 WHERE entry=15931;
+-- Razuvious
+UPDATE creature_template SET SpeedRun=1.68, Detection=35 WHERE entry=16061;
+-- Loatheb
+UPDATE creature_template SET Detection=35 WHERE entry=16011;
```
Comment (insunaa) [inline Updates/0414_naxxramas_boss_fixes.sql:38]: Ahh, forgot I already made Leashing updates for some, let me just combine them.

**Comment by cala** [comment]
Merged, thank you very much ! 👍

**Pull request merged**